Update README.md (#138)
PsiQ/qref: README.md

pip install .
```

### Graphviz

> For visualization we use `graphviz`, which requires the [Graphviz](https://graphviz.org/) system package to be installed on your machine — the Python `graphviz` library alone is not sufficient.

Reviewer: ```suggestion
For visualization we use `graphviz` Python package, which requires the [Graphviz](https://graphviz.org/) executable to be available in your PATH — the Python package is not sufficient.
```

> - **macOS:** `brew install graphviz`

> > - **Linux (Debian/Ubuntu):** `sudo apt-get install graphviz`

Reviewer: **suggestion**: It's probably  better to recommend usage of `apt` instead of `apt-get`. Also, I'd recommend we add Fedora instructions as well.

```suggestion
> - **Linux (Debian/Ubuntu):** `sudo apt install graphviz`
> - **Linux (Fedora):** `sudo dnf install graphviz`
```
